### PR TITLE
Scrub sample info, add some pages and a single post for the previous meetup

### PR DESCRIPTION
**Pull request by manicminer**
@cobusbernard @adrianmoisey 

Feedback welcomed on the content. I pulled from various other CoCs for the code of conduct.

### CODE REVIEW

**Comment by manicminer** [comment]
@juliaquickfall sorry forgot to mention

File: _config.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Site settings
 title: DevOps Cape Town
-email: team@devops.capetown
+email: organisers@devops.capetown
 description: >
-  DevOps Cape Town is awesome much lots.
+  DevOps Cape Town is a community group interested in advancing the adoption and evolution of DevOps in our home city
```
Comment (adrianmoisey) [inline _config.yml]: Should that be "... in their home city" ?

**Comment by adrianmoisey** [comment]
Damn Tom, that's some good work. Seems legit :+1:

File: _config.yml (hunk 1 above)
Comment (manicminer) [inline _config.yml]: @adrianmoisey Well spotted, will change.

**Comment by manicminer** [comment]
Great! Thanks for the feedback, I'll go ahead and merge it.

**Pull request merged**